src/types.ts

```diff
@@ -83,6 +88,9 @@ export interface HoneycombOptions extends Partial<WebSDKConfiguration> {
    */
   debug?: boolean;
 
+  /** Any additional attributes to add to the resource */
+  resourceAttributes?: ResourceAttributes;
```

Comment:
I like that we're naming it `resourceAttributes` because that's what it is, but I also acknowledge your earlier point about people maybe not understanding what it means. Maybe we can spiff up the comment a bit to say "`Any additional attributes to be included on all spans in a trace`" or something to that effect?

Reply:
Maybe "included on all data"? Since if this SDK ever starts emitting logs or metrics that'll still be true.